spacy/tokens/span.pxd

```diff
@@ -7,19 +8,21 @@ from ..structs cimport SpanC
 
 cdef class Span:
     cdef readonly Doc doc
-    cdef SpanC c
+    cdef shared_ptr[SpanC] c
     cdef public _vector
     cdef public _vector_norm
 
     @staticmethod
-    cdef inline Span cinit(Doc doc, SpanC span):
+    cdef inline Span cinit(Doc doc, const shared_ptr[SpanC] &span):
         cdef Span self = Span.__new__(
             Span,
             doc,
-            start=span.start,
-            end=span.end
+            start=span.get().start,
+            end=span.get().end
         )
         self.c = span
         return self
 
     cpdef np.ndarray to_array(self, object features)
+
+    cdef SpanC *span_c(self)
```

Comment:
```suggestion
    cdef SpanC* span_c(self)
```

And all the similar cases in the rest of PR, to match our style (which probably isn't documented formally anywhere, not something you would have necessarily known either way).

Reply:
That's a very un-C'ish style. But I am fine with making those changes for consistency.

Reply:
Done!

Reply:
I know, and I have no idea why it ended up that way, but that's how it currently is.

Reply:
Seems like the Cython docs also use this style. Maybe it's standard for Cython?

Still hoping for a great Cython formatter :).